src/components/domain/auth/signup/SignupModal.tsx

```diff
@@ -19,10 +22,29 @@ import {
   FormMessage,
 } from '@/components/ui/form';
 
-export default function SignupModal() {
+interface SignupModalProps {
+  onSuccess?: () => void;
+  onClose?: () => void;
+}
+
+export default function SignupModal({ onSuccess, onClose }: SignupModalProps) {
   const id = useId();
   const isSmallScreen = useMediaQuery('(max-width: 430px)');
   const [isAgreed, setIsAgreed] = useReducer((state) => !state, false);
+  const [isButtonDisabled, setIsButtonDisabled] = useState(false);
```

Comment:
다음에 파일 수정해주실 때 타입 명시해주시면 감사하겠습니다!

```suggestion
  const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(false);
```

Reply:
아 `useState` 타입이 다 빠져 있었네요. 추가했습니다!